_opensearch/snapshots/snapshot-restore.md

```diff
@@ -1,23 +1,21 @@
 ---
 layout: default
 title: Take and restore snapshots
-nav_order: 65
+parent: Snapshots
+nav_order: 10
+has_children: false
 ---
 
 # Take and restore snapshots
 
-Snapshots are backups of a cluster's indices and state. State includes cluster settings, node information, index metadata (mappings, settings, templates, etc.), and shard allocation.
-
-Snapshots have two main uses:
-
-- **Recovering from failure**
-
-  For example, if cluster health goes red, you might restore the red indices from a snapshot.
+Snapshots aren't instantaneous. They take time to complete and do not represent perfect point-in-time views of the cluster. While a snapshot is in progress, you can still index documents and make other requests to the cluster, but new documents and updates to existing documents generally aren't included in the snapshot. The snapshot includes primary shards as they existed when OpenSearch initiated the snapshot. Depending on the size of your snapshot thread pool, different shards might be included in the snapshot at slightly different times.
 
-- **Migrating from one cluster to another**
+OpenSearch snapshots are incremental, meaning that they only store data that has changed since the last successful snapshot. The difference in disk usage between frequent and infrequent snapshots is often minimal.
 
-  For example, if you're moving from a proof-of-concept to a production cluster, you might take a snapshot of the former and restore it on the latter.
+In other words, taking hourly snapshots for a week (for a total of 168 snapshots) might not use much more disk space than taking a single snapshot at the end of the week. Also, the more frequently you take snapshots, the less time they take to complete. Some OpenSearch users take snapshots as often as every half hour.
```

Comment:
Instead of "half hour", can we use "30 minutes"?

Reply:
Done